queue/queue.py

```diff
@@ -3,13 +3,19 @@ def __init__(self):
     self.size = 0
     # what data structure should we
     # use to store queue elements?
-    self.storage = 
+    # linked list
+    self.storage = []
 
   def enqueue(self, item):
```

Comment:
You might want to look at using something besides an array adding to the front is going to be quite slow.